.codecov.yml

```diff
@@ -14,6 +14,8 @@ component_management:
         # The minimum coverage ratio to send a success status is the base
         # commit coverage (pull request base or parent commit).
         target: auto
+        # Allow the coverage to drop by this percentage.
+        threshold: 1%
```

Comment:
@nahueespinosa can we have a strict lower bound? Like, all good with small drops but going below 95% (or 90%, whichever we prefer) is a no-no.

Reply:
AIUI, looking at https://docs.codecov.com/docs/commit-status, we can't do both checks at the same time.
In https://github.com/Ekumen-OS/beluga/pull/353/commits/2d15264375cf2ea704166d374c1828aa9c618ee2, I changed the configuration to have a strict lower bound for project level coverage, and use the target branch as reference in patch level coverage (with a 1% threshold). I think this might work.

Reply:
IMO, leaving only a lower limit without patch verification may result in a slow decrease towards that limit, and people having to add coverage for the old code in order to unblock themselves.

Reply:
> IMO, leaving only a lower limit without patch verification may result in a slow decrease towards that limit, and people having to add coverage for the old code in order to unblock themselves.

That's a very good point.